src/pykiso/test_result/multi_result.py

```diff
@@ -0,0 +1,239 @@
+        """Initialize the result classes with the parameters passed in arguments."""
+        self.result_classes = [
+            result(*args, **kwargs)
+            if isinstance(result, XmlTestResult)
+            else result(*self._get_arguments_bannertestresult(*args, **kwargs))
+            for result in self.result_classes
+        ]
+        return self
+
+    def _get_arguments_bannertestresult(
+        self, *args, **kwargs
+    ) -> List[TextIO, bool, int]:
+        """Function to get the argument for the BannerTestResult initialisation
+        since it only takes 3 arguments and more can be passed.
+        """
+        list_arguments = []
+        args = list(args)
+        for name in signature(BannerTestResult).parameters.keys():
+            arg = kwargs.get(name)
+            list_arguments.append(arg if arg is not None else args.pop(0))
+        return list_arguments
+
+    def __getattr__(self, name: str) -> Any:
+        """Function implemented to get the attributes such as error, failures,
+            sucesses,expectedFailures.
+
+        :param name: name of the attribute
+        """
+        return getattr(self.result_classes[0], name)
+
+    @property
+    def error_occurred(self) -> Optional[bool]:
+        """Return if an error occured for a BannerTestResult"""
+        for result in self.result_classes:
+            if hasattr(result, "error_occurred"):
+                return result.error_occurred
+
+    def startTest(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the startTest function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.startTest(test)
+
+    def startTestRun(self) -> None:
+        """Call the startTestRun function for all result classes."""
+        for result in self.result_classes:
+            result.startTestRun()
+
+    def stopTestRun(self) -> None:
+        """Call the stopTestRun function for all result classes."""
+        for result in self.result_classes:
+            result.stopTestRun()
+
+    def stop(self) -> None:
+        """Call the stop function for all result classes."""
+        for result in self.result_classes:
+            result.stop()
+
+    def stopTest(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the stopTest function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.stopTest(test)
+
+    def addSuccess(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the addSuccess function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.addSuccess(test)
+
+    def addSkip(self, test: Union[BasicTest, BaseTestSuite], reason: str) -> None:
+        """Call the addSkip function for all result classes.
+
+        :param test: running testcase
```

Comment:
reason param is not describe in the docstring.

Reply:
Done.